cypress/e2e/api.cy.js

```diff
@@ -0,0 +1,17 @@
+/// <reference types="Cypress" />
+
+import 'cypress-plugin-api'
+
+
+
+describe('GET customers', ()=> {
+    it('Success status', ()=>{
+        cy.api({
+            method: 'GET',
+            url: 'API_URL',
```

Comment:
Aqui deveria ser algo assim:

```suggestion
            url: `${Cypress.env('API_URL')}/customers`,
```